### PR TITLE
Fixed width of read more button in card

### LINKED ISSUE #7431: Fix the spacing in the read more button
The read more button under the methods url has a spacing issue.
The padding/margin needs to be fixed so that the read more text doesn't overflow out of the button.

> What did you expect to see that you didn't?
image

### Please show us where to look

https://publiclab.org/methods

### PR DESCRIPTION
**Pull request by Tlazypanda**
Fixes #7431 

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

# Screenshots -

Screenshot from 2020-02-17 12-52-50

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@publiclab/comment-maintainers @jywarren @cesswairimu can you kindly review? Thanks :v:

File: app/views/notes/_card.html.erb [hunk 1]
```diff
@@ -53,9 +53,9 @@
         <li><a>Total Likes: <%= node.likers.length %></a></li>
         <div class="content" style="width: 100%" >
           <% if @compact.nil? %>
-           
+
             <li class="buttons mb-2"><% if @current_user && ['admin','moderator'].include?(@current_user.role) %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %></li>
-            <li class="buttons"><a style="width: 90px;" class="btn btn-outline-secondary" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a></li>
+            <li class="buttons"><a style="width: 100px;" class="btn btn-outline-secondary" href="<%= node.path %>"><%= t('notes._notes.read_more') %></a></li>
```
Comment (VladimirMikulic) [inline app/views/notes/_card.html.erb]: Setting a fixed width is the problem here.
Screenshot_20200209_102411

The text is not centred. The inline style should be removed.

**Comment by cesswairimu** [comment]
Hey @Tlazypanda, thanks for working on this...Did you checkout @VladimirMikulic's review above? What do you think? Thanks

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu @VladimirMikulic Thanks for the review :smile: since the inline styling was already in place I decided to modify it so that I don't break the code anywhere.I have updated the pr now , kindly review :sweat_smile: also should I open another issue to lose the inline styling in some templates ...let me know what you think about this.Thanks :v:

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu @VladimirMikulic I think travis failing is not associated with my changes,can you kindly check the same? Thanks a lot! :v:

**Comment by VladimirMikulic** [comment]
Could you post a screenshot of the new change?

**Comment by Tlazypanda** [comment]
@VladimirMikulic updated screenshot in the pr description.Thanks :smile:

**Comment by Tlazypanda** [comment]
@jywarren is this ready to be merged or more changes required? 😅

**Comment by cesswairimu** [comment]
Great, restarting travis

**Comment by Tlazypanda** [comment]
@cesswairimu should I make nay change to make travis pass? not really sure :sweat_smile:

**Comment by VladimirMikulic** [comment]
@Tlazypanda Travis is not your fault :)
The best that you can do is to either force push to restart it or wait for @cesswairimu to restart it.

**Pull request merged**